scripts/externalize_js.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/sh
+
+# PURPOSE:
+# Externalizing JS is a fairly manual process that has to happen
+# on a couple hundred files. Hopefully this will simplify that process
+
+# For now, it creates the new file, places whatever was in the js-inline
+# script tags in the new file, and adds the new necessary static import
+# to the html file.
+
+# TODOS:
+#   check for template tags in the js file
+#       print necessary initial page data
+#       actually add initial page data tags
+#       convert tags to requesting initial page data
+#   auto indent the moved javascript
+
+# strict mode --> kill it if something fails
+set -euo pipefail
+
+app=$1
+module=$2
+
+# formulate locations and names
+html_file_location="./corehq/apps/$app/templates/$app/$module.html"
+new_module_name="$app/js/$module"
+new_module_location="./corehq/apps/$app/static/$app/js/$module.js"
```

Comment:
What happens if this file already exists?

Reply:
That's a good point. For now, it just adds the new stuff to the bottom. 

Perhaps it should complain and ask whether or not to continue?

Reply:
Complaining might be nice, yeah, would make it obvious that there's a bit more manual work to do.